### DISCUSSION TITLE
Tutorial - Blockman - How to show only focused block

### DISCUSSION THREAD

**Discussion by leodevbro** [leodevbro/vscode-blockman#111] [General]
Image:
![image](https://user-images.githubusercontent.com/54504880/231464294-611ef053-0d71-4b5d-b69a-cc51cadd5b7e.png)

Settings:
```jsonc
// settings.json (User/Global config, not Workspace config)
// To open this file in VS Code, press F1, type 'settings json' and choose 'Preferences: Open Settings (JSON)'
{
  // ...
  "blockman.n33A01B1FromDepth0ToInwardForAllBorders": "10,0,0,1; empty",
  "blockman.n33A01B2FromDepth0ToInwardForAllBackgrounds": "10,0,0,1; empty",

  "blockman.n33A02B1FromFocusToOutwardForAllBorders": "!",
  "blockman.n33A02B2FromFocusToOutwardForAllBackgrounds": "!",

  "blockman.n33A03B1FromDepth0ToInwardForFocusTreeBorders": "!",
  "blockman.n33A03B2FromDepth0ToInwardForFocusTreeBackgrounds": "!",

  "blockman.n33A04B1FromFocusToOutwardForFocusTreeBorders": "!",
  "blockman.n33A04B2FromFocusToOutwardForFocusTreeBackgrounds": "!",

  "blockman.n33A05B1FromFocusToInwardForAllBorders": "50,0,0,0; basic",
  "blockman.n33A05B2FromFocusToInwardForAllBackgrounds": "50,0,0,0; basic"
}

```

**Comment by leodevbro** [comment]
So, if you display only focused block, then I guess you will want to enable native indent guides (horizontal lines) of VS Code.

![image](https://user-images.githubusercontent.com/54504880/231467853-03baa5a1-c7ec-4b1d-ac78-5b13f9914f32.png)

**Reply by bigbasti** [reply]
How do you get the indent guides (horizontal lines) of VS Code back? I tried the config above and im missing them

**Reply by leodevbro** [reply]
> How do you get the indent guides (horizontal lines) of VS Code back? I tried the config above and im missing them

I think you can use VS Code settings:
```
"editor.guides.indentation": true,
"editor.guides.bracketPairs": true,
```

Also, with Blockman, when Blockman is enabled and you want to disable Blockman and get back the lines,
you can press `F1` and search `Blockman disable` and you will have two commands:
->`Blockman Toggle Enable/Disable And Force Show/Hide Indent Guides` -> this is probably what you need.
->`Blockman Toggle Enable/Disable`

<img width="942" height="140" alt="image" src="https://github.com/user-attachments/assets/4bff6b82-e7dd-4e61-968f-cdcb3583236a" />
